# Common parameters
wheelbase: 0.5 # Car wheelbase (in m)
width: 0.3 # Car width (in m)
length: 0.7 # Car length (in m)
distance_tolerance: 1 # Minimum distance error to consider that the planner has reached the goal (in meters) was 0.5
angle_tolerance: deg(360) # Minimum angle error to consider that the planner has reached the goal (in radians) was 15 deg

speed_min: -1.5 # Minimum speed (in meters/seconds) was -0.6
speed_max: 1.5 # Maximum speed (in meters/seconds) was 0.6
steering_angle_min: -1.57 # (was -pi/4) Minimum angular speed (in radians/seconds)
steering_angle_max: 1.57 # (was pi/4) Maximum angular speed (in radians/seconds)

publish_frequency: 10 # ROS path publish frequency (in Hz)
path_frame: 'map' # ROS frame where to publish the path

# MPC parameters
N: 100 # Prediction horizon (in steps) was 20
R_speed: 1. # Weight on the speed (input)
R_steering_angle: 1. # Weight on the steering_angle (input)
kappa: 1. # Weight on the total time
